pkg/opencensus/init.go

```diff
@@ -0,0 +1,46 @@
+package opencensus
+
+import (
+	"log"
+
+	"contrib.go.opencensus.io/exporter/jaeger"
+	prometheus_exporter "contrib.go.opencensus.io/exporter/prometheus"
+	pb "github.com/buildbarn/bb-storage/pkg/proto/configuration/bb_storage"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/trace"
+	"go.opencensus.io/zpages"
+)
+
+// Initialize sets up Opentracing with Jaeger and a Prometheus exporter.
+func Initialize(configuration *pb.JaegerConfiguration, namespace string) {
```

Comment:
This function is likely not any different from the copy that's already in bb-storage, right? If so, be sure to throw this out and cal the one from bb-storage. bb-storage is intended to act as a utility library for the rest of bb-* anyway.